pkgs/tools/networking/zerotierone/default.nix

```diff
@@ -41,12 +34,15 @@ in stdenv.mkDerivation {
     substituteInPlace ./make-linux.mk \
       --replace '-march=armv6zk' "" \
       --replace '-mcpu=arm1176jzf-s' ""
+
```

Comment:
can you add a comment that links back to more context, like the upstream issue? All the rest looks good.

Reply:
Huh? I don't think there is an upstream issue here? I'm of the impression this is a packaging issue here and not an upstream issue. However, I can add some more details to clarify why it's needed.

Reply:
@zimbatm does that work?

Reply:
thanks, it will help with the next release